testutil/crypto/crypto.go

```diff
@@ -31,10 +32,16 @@ func NewCryptoIdentityFromBytesSeed(seed []byte) CryptoIdentity {
 	return CryptoIdentity{PV: privKey}
 }
 
-func NewCryptoIdentityFromIntSeed(i int) CryptoIdentity {
-	iUint64 := uint64(i)
+func NewCryptoIdentityFromRandSeed() CryptoIdentity {
+	b := make([]byte, 8)
+	_, _ = rand.Read(b)
+	seed := binary.BigEndian.Uint64(b)
```

Comment:
Nice! I forgot to add this. I think you can do rand.Uint64() directly

https://pkg.go.dev/math/rand#Uint64